app/src/main/java/ca/ualberta/CMPUT3012019T02/alexandria/model/message/ImageMessage.java

```diff
@@ -6,18 +6,24 @@
 
 public class ImageMessage extends Message {
 
-    public ImageMessage(String content, String status, Date date, String sender) {
-        super("Image", content, status, date, sender);
+    public ImageMessage(Bitmap content, String status, Date date, String sender) {
+        if (!(content instanceof Bitmap.class)) {
+            throw new IllegalArgumentException("Content needs to be a Bitmap object");
+        }
+
+        super("image", content, status, date, sender);
     }
 
     public Bitmap getImage() {
-        // TODO Finish implementation
-        throw new UnsupportedOperationException();
+        return this.content;
```

Comment:
You cannot convert dirrectly

Reply:
You need a few lines to serialize, but we don't need this functionality yet

Reply:
See https://stackoverflow.com/a/34165515
https://stackoverflow.com/a/11654825
If you want to implement now if not that is fine